### PR TITLE
docs: enable search in upstream docs website - fixes #3830

### LINKED ISSUE #3830: Enable search in upstream docs website
The search feature does not work in https://crc.dev/crc/, because the official Antora container image does not include the [Antora Lunr extension](https://gitlab.com/antora/antora-lunr-extension/). Create our own container based on the offical Antora image, with the addition of the Lunr extension, host it on Quay, and use it.

### PR DESCRIPTION
**Pull request by themr0c**
chore: enable search in upstream docs website

Also fixes YAML indentation. Other files in the repository are using a 2 spaces indentation. 

fixes #3830

Screenshot_20230911_142334

### CODE REVIEW

File: Makefile
Comment (praveenkumar) [inline Makefile]: if that works can we push that image to `crcont` org in quay?

Comment (themr0c) [reply Makefile]: Yes of course! Can you give me access?

**Comment by praveenkumar** [comment]
Can we squash all 3 commits to single one because last 2 commits are just fixing the mistake/suggestion to first one.

**Review by gbraad — APPROVED** [review]
/approve when squashed

**Comment by themr0c** [comment]
I launched a retest for ci/prow/e2e-crc, but I believe the test failure is unrelated to the pull request.

**Comment by themr0c** [comment]
@gbraad @praveenkumar Can you help me move forward: 

- Failed tests: The e2e test is failing, for reasons unrelated to this pull request. It blocks the merge. How can we move forward?
- Rebase, squash: I am used to use GitHub web UI features to rebase, and squash and merge. I don't see them here. I am thinking it might be because I have insufficient rights. How do you proceed usually in  this repository?

**Comment by praveenkumar** [comment]
>Failed tests: The e2e test is failing, for reasons unrelated to this pull request. It blocks the merge. How can we move forward?

Ignore the failure since it is unrelated to this PR.

> Rebase, squash: I am used to use GitHub web UI features to rebase, and squash and merge. I don't see them here. I am thinking it might be because I have insufficient rights. How do you proceed usually in this repository?

We disabled it from github web UI so better to do it locally using `git rebase -i HEAD~3` and repush it, if you are unfamiliar then I can try to do it myself and push it.

**Comment by themr0c** [comment]
Rebased and squashed and lost LGTM label.

**Comment by praveenkumar** [comment]
merged with rebase

/close

**Pull request closed without merging**
